tests/test_config.py

```diff
@@ -32,19 +33,39 @@
 })
 
 
-def _compare_configs(self, dict_config, server_config):
-    """Compare a server config in two different forms.
+def _convert_bsc_attrs(bsc_attrs):
```

Comment:
"bsc" == "BaseServerConfig"

I had trouble coming up with a name for this method.